src/Pmi/EntityManager/DoctrineRepository.php

```diff
@@ -1,7 +1,7 @@
 <?php
 namespace Pmi\EntityManager;
 
-class DoctrineRepository
+class DoctrineRepository extends EntityManager
```

Comment:
I see what you're doing here, but I don't think we should be making this extend from the EntityManager class because they are not analogous.  The entity manager is a level above a specific entity repository and this inheritance can get confusing.

One way to accomplish what you have done here would be to move the timestamp parsing and timezone conversion methods to the `Util` class as static methods (and add the timezone as a method parameter).  Then, you can still have your `fetchAll` method in the entity manager, but the mismatched inheritance won't be necessary.

The `dateTimesToStrings` method is only used by the `DoctrineRepository` `insert` and `update` methods, so that can stay in the `DoctrineRepository` class.